klog.go

```diff
@@ -800,10 +800,16 @@ func kvListFormat(b *bytes.Buffer, keysAndValues ...interface{}) {
 			v = missingValue
 		}
 		b.WriteByte(' ')
-		if _, ok := v.(fmt.Stringer); ok {
+
+		switch v.(type) {
+		case string, ObjectRef:
```

Comment:
```suggestion
		case string:
```
`ObjectRef` should be handle by stringer.